src/components/Navbar.js

```diff
@@ -44,7 +44,8 @@ export class Navbar extends Component {
         <Navigation color="light" light expand="md" className="custom-max-width">
           <NavbarBrand href="/">OCL for OpenMRS</NavbarBrand>
           <NavbarToggler onClick={this.toggle} />
-          {loggedIn && (
+          {
```

Comment:
Is this a formatting change?

Reply:
@Duncanian, there has been an issue everyone is faceing from travis that makes us make this change.